packages/utxo-lib/tests/derivation.test.ts

```diff
@@ -0,0 +1,41 @@
+import fixtures from './__fixtures__/derivation';
+import { deriveAddresses } from '../src/derivation';
+import { testnet, regtest } from '../src/networks';
+
+const getNetwork = (network?: string) => {
+    switch (network) {
+        case 'testnet':
+            return testnet;
+        case 'regtest':
+            return regtest;
+        default:
+            break;
+    }
+};
+
+describe('Testing address derivation from xpubs', () => {
+    fixtures.derivation.forEach(f => {
+        it(f.description, () => {
+            const { xpubs, change, receive, pathPrefix } = f;
+            const network = getNetwork(f.network);
```

Comment:
nitpicking: Why not adding `network` to the destructing object above for consistency?

Reply:
Because there were two different things which should be called `network`. Renamed to `symbol`.
https://github.com/trezor/trezor-suite/pull/4907/files#diff-968f4ae492bd4dd25691c904df1234890956caf39e66cde58d86b1d61658b2d1R20